media_kit_video/lib/media_kit_video_controls/src/controls/widgets/video_controls_theme_data_injector.dart

```diff
@@ -0,0 +1,98 @@
+/// This file is a part of media_kit (https://github.com/media-kit/media-kit).
+///
+/// Copyright © 2021 & onwards, Hitesh Kumar Saini <saini123hitesh@gmail.com>.
+/// All rights reserved.
+/// Use of this source code is governed by MIT license that can be found in the LICENSE file.
+import 'package:flutter/widgets.dart';
+
+import 'package:media_kit_video/media_kit_video_controls/src/controls/cupertino.dart';
+import 'package:media_kit_video/media_kit_video_controls/src/controls/material.dart';
+import 'package:media_kit_video/media_kit_video_controls/src/controls/material_desktop.dart';
+
+/// {@template video_controls_theme_data_injector}
+///
+/// VideoControlsThemeDataInjector
+/// ------------------------------
+///
+/// Makes the various `*VideoControlsTheme` [InheritedWidget]s from provided [context] to current [BuildContext] and its descendants.
+///
+/// The fallback to default values is done by [VideoControlsThemeDataInjector] itself.
+///
+/// * [CupertinoVideoControlsTheme]
+/// * [MaterialVideoControlsTheme]
+/// * [MaterialDesktopVideoControlsTheme]
+///
+/// {@endtemplate}
+class VideoControlsThemeDataInjector extends StatefulWidget {
```

Comment:
Removed `ControlsThemeDataBuilderInheritedWidget` in favor of `VideoControlsThemeDataInjector`.